assets/sass/v2/_consent-email.scss

```diff
@@ -15,11 +15,41 @@
   }
 }
 
+.enduser-email-otp-only {
+  .o-form-button-bar {
+    display: flex;
+  }
+  &--info {
+    display: flex;
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
+    align-items: center;
+  }
+  &--icon {
+    margin-right: 1rem;
+    height: 2rem;
+    width: 2rem;
+    background-size: cover;
+  }
+}
```

Comment:
Added some new css classes here for potential more changes on the otp-only page that is different from consent page.